### PR TITLE
Project Details

### PR DESCRIPTION
**Pull request by miltonHenschel**
- [ ] Load project section dynamically (from JavaScript instead of html)
- [ ] Open pop up window showing project details when see project button is clicked
- [ ] Close pop up window when close button is pressed
- [ ] Email (form) validation

### CODE REVIEW

File: js/projects.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    </div>
+    <div id="div-section">
+      <div class="div-menu-img"><img id="div-section-img" src="${projects[button.id].image}" alt="" /></div>
+      <div id="div-section-menu">
+        <div class="div-menu"><p id="div-section-p">${projects[button.id].description}</p></div>
+        <div id="div-menu-last">
+          <a id="div-menu-live" href="${projects[button.id].live_version}"><button class="div-menu-button">See Live&nbsp;<div><img src="./images/social-media-header/live.png" alt="live icon" /></div></button></a>
+          <a id="div-menu-source" href="${projects[button.id].source_code}"><button class="div-menu-button">See Source&nbsp;<div><img src="./images/social-media-header/github.png" alt="" /></div></button></a>
+        </div>
+      </div>
```
Comment (Abenezer-Tilahun) [inline js/projects.js:208]: - [ ] As stated the project requirement  You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly Adjust The popup to make it responsive, to fits desktop sizes properly.

Your Desktop size Popup Look likes this : 
Capture

The Figma Desktop size Popup Look likes this : 
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @miltonHenschel   , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional  README 
✔️ Good commit messages

almost

## Required Changes ♻️

 _Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/projects.js (hunk 1 above)
Comment (Owusu-Desmond) [inline js/projects.js:208]: - [x] Please as requested by the previous code reviewer, from the project requirements, you should stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=YHGkZAlalTNDuhZx-0) as much as possible. But as you can see you have some differences in the design. 

You can see the differences below your modal design and the Figma design do not match :-1:  


| Your Design |Figma Design (Have margins around) |
| :---:   | :---: | 
| Screenshot from 2022-11-26 03-55-08 | image |

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @miltonHenschel  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/desktop.css [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+
+  /* PROJECT DETAILS */
+  .section-menu {
+    left: 1rem;
+    right: 1rem;
+    top: 1rem;
+    bottom: 1rem;
+    display: flex;
+    flex-direction: column;
+    padding-top: 0.5rem;
```
Comment (OyePriscilla) [inline css/desktop.css]: - [ ] [optional] Please kindly increase the padding a bit so as to have something very close to the design. Following design template to details shows that we can perfectly deliver whatsoever given to us as developers. Thanks. 😄

**Review by OyePriscilla — APPROVED** [review]
Hi @miltonHenschel 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Descriptive pull request message and title ✔️
Linter error checks validated ✔️
Commit messages are meaningful. ✔️
GitHub flow observed ✔️

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/desktop.css (hunk 2 above)
Comment (miltonHenschel) [reply css/desktop.css]: Thanks for the review. I will do that.

**Pull request merged**